sdk/spring/azure-spring-boot/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## 3.7.0-beta.1 (Unreleased)
 
+## 3.6.1 (2021-07-02)
+### New Features
+- Upgrade to [spring-boot-dependencies:2.5.2](https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.5.2/spring-boot-dependencies-2.5.2.pom).
```

Comment:
We need to add changelog to azure-spring-boot-starter as well.

Reply:
Already added.